src/test/java/jenkins/plugins/office365connector/CardBuilderTest.java

```diff
@@ -505,4 +506,54 @@ public void getEscapedDisplayName_OnNameWithSpecialCharacters_EscapesSpecialChar
         // then
         assertThat(displayName).isEqualTo("this\\_is\\_my\\-very\\#special \\*job\\*");
     }
+
+    @Test
+    public void getCardThemeColor_OnSuccessResult_ReturnsGreen() {
+        Result successResult = Result.SUCCESS;
+        String themeColor = Deencapsulation.invoke(CardBuilder.class, "getCardThemeColor", successResult);
+        
+        String greenColorString = String.format("#%06X", Color.GREEN.getRGB() & 0xFFFFFF);
```

Comment:
I think this is to complicated to get green color :)

Reply:
I agree, I will use the actual string instead of that method.